docker/build_scripts/build_utils.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+# Helper utilities for build
+
+PYTHON_DOWNLOAD_URL=https://www.python.org/ftp/python
+OPENSSL_DOWNLOAD_URL=http://www.openssl.org/source
+GET_PIP_URL=https://bootstrap.pypa.io/get-pip.py
+
+
+function check_var {
+    if [ -z "$1" ]; then
+        echo "required variable not defined"
+        exit 1
+    fi
+}
+
+
+function pyver_ge {
+    # Echo 1 if first python version is greater or equal to second
```

Comment:
I think this should work (sort options set up to numerically compare first 2 "." separated fields; it's a stable sort so if they match the output will be unchanged, making it a `<=` operation):
```
major_minor_version_lte() {
  echo "$1\n$2" | sort -n -t "." -k 1,1 -k 2,2 -s | head -n 1 | grep -q "$1"
}
```